README.md

```diff
@@ -376,12 +376,12 @@ First we have to resolve the conflict by hand.
 If you open `Bob.txt` you'll see something similar to this (I've truncated whatever we might have put on the second line before): 
 
 ```Diff
-<<<<<<< HEAD
-Hi! I'm Bob. I've been here for a while now.
-=======
-Hi! I'm Bobby. I'm new here.
->>>>>>> bobby_branch
-[... whatever you've put on line 2]
+ <<<<<<< HEAD
```

Comment:
Github wasn't letting me finish the merge, so I had to escape the diff special sequences. Maybe you know an elegant solution to this problem?

Reply:
Sorry for the delay :( 

It looks to me that the issue is just with one conflict that actually happens because of the typo pull request I've accepted a while ago, that part which seems like a conflict diff can just be ignored. 

I've just tried it out locally, by cloning your repo, but to actually get it pushed there it's probably the easiest if you do it. (Sounds easier than me making a fork and PR to your repo) 

So what I've done: 
* set up my repo as a remote, if you haven't already done that
` git remote add upstream git@github.com:UnseenWizzard/git_training.git`
* `git fetch upstream`
* IMHO the cleanest thing is to `rebase` your `patch-1` on the upstream master instead of merging, then github also can't complain about being unable to rebase
So `git rebase upstream/master` 
* There I had one actual conflict, towards the end of the file, easy to resolve
* Ignore the diff that's part of the tutorial, add, continue the rebase, push, done :)